util/cli/cli_test.go

```diff
@@ -29,3 +36,100 @@ func TestPromptPassword_Fallback(t *testing.T) {
 	password := PromptPassword("")
 	require.Equal(t, pwd, password)
 }
+
+func TestSetLogFormat(t *testing.T) {
+	tests := []struct {
+		name          string
+		logFormat     string
+		expected      string
+		expectedFatal bool
+	}{
+		{
+			name:      "log format is set to json",
+			logFormat: "json",
+			expected:  "json",
+		},
+		{
+			name:      "log format is set to text",
+			logFormat: "text",
+			expected:  "text",
+		},
+		{
+			name:      "log format is not set",
+			logFormat: "text",
+			expected:  "text",
+		},
+		{
+			name:          "invalid log format",
+			logFormat:     "invalid",
+			expected:      "",
+			expectedFatal: true,
+		},
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			if tt.expectedFatal {
+				if os.Getenv("TEST_FATAL") == "1" {
+					SetLogFormat(tt.logFormat)
+					return
+				}
+				cmd := exec.Command(os.Args[0], "-test.run="+t.Name())
```

Comment:
Are we using exec.Command to test except here?

Reply:
Thanks!